twitter.html

```diff
@@ -0,0 +1,327 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="container">
+<!-- LEFT MENU -->
+        <div class="left-menu">
+
+            <a class="twitter-logo" href="#"><img src="./images/SVG's/twitter-color-svgrepo-com.svg" alt="Twitter logo"
+                    width="30px"></a>
+
+            <svg fill="#1DA1F2" height="26.25px" width="26.25px" version="1.1" xmlns="http://www.w3.org/2000/svg"
```

Comment:
Did using this fill attribute work for you?

Reply:
On some SVG's not all.